### ISSUE TITLE
Dealing with so many open PRs

### ISSUE THREAD

**Issue by mariuszskon** [rg3/youtube-dl#15045]
### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [x] Other

---

So it appears that youtube-dl is, well, [struggling with a pull request overload](https://github.com/rg3/youtube-dl/pulse/monthly). This issue is to discuss possible solutions for this. I am not trying to start any controversy, just to make everyone aware of the problem.

I believe that in a large part, there are two main sub-issues here:

1. Not enough maintainers. @dstftw is the only really active maintainer, and he cannot be expected to do almost all of the work.

2. Plenty of old PRs that should have been closed or merged by now. See #5177 and #5163 for PRs that are surely dead, or #7080 and #14202 for ones where the submitter has received no response from maintainers for quite a long time.

I have no suggested solutions for this, so I wish to get some discussion going.

**Comment by sleske** [comment]
Yes, my thought exactly. I submitted a PR (#14598) that seems to have been ignored completely. It's frustrating to submit PRs and not even get a one-line response.

At the same time, I'm willing to volunteer to review PRs, if that is desired.
Maybe the project can establish a list of "PR reviewers" or "PR triagers" who can check incoming PRs. These people can work without full commit/merge rights at first, but could e.g. tag PRs that look fine for final review and merge (tag "ready-to-merge"), or they could approve a PR in a [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/). Then a "full member" with write access can just do a quick final check & merge.

**Comment by WPFilmmaker** [comment]
My two cents as non-dev, why not spending a period (a week?) just focusing on PRs ?

**Comment by gardenappl** [comment]
Ironically, this very PR is also dead.

**Comment by mariuszskon** [comment]
I was considering starting a "proper" fork which is more aggressive in terms of accepting new extractor pull requests, but I am afraid it will get out of hand quickly.

**Comment by gardenappl** [comment]
Have you thought about doing [something like this](https://gist.github.com/jancborchardt/6155185#clean-up-inactive-issues)?



```
I’m closing this issue because it has been inactive for a few months. This probably means that it is not reproducible or it has been fixed in a newer version. If it’s an enhancement and hasn’t been taken on for so long, then it seems no one has the time to implement this.

Please reopen if you still encounter this issue with the [latest stable version](http://owncloud.org/install/) and then please use the [issue template](https://raw.github.com/owncloud/core/master/issue_template.md). You can also contribute directly by providing a patch – see the [developer manual](http://doc.owncloud.org/server/9.0/developer_manual/). :)

Thank you!
```

Obviously I'm not a youtube-dl maintainer, just throwing my 2 cents, waiting for [my own PR to get reviewed](https://github.com/ytdl-org/youtube-dl/pull/26394). :D
